src/Test/QuickCheck.hs

```diff
@@ -50,8 +50,8 @@ them to:
 +++ Ok, passed 100 tests.
 @
 
-That's because GHCi will default any type variables in your property to '()', so in the example
-above @quickCheck@ was really testing that '()' is equal to itself. To avoid this behaviour it
+That's because GHCi will default any type variables in your property to @()@, so in the example
+above @quickCheck@ was really testing that @()@ is equal to itself. To avoid this behaviour it
```

Comment:
`()` is not actually in the Haddocks for Prelude, so linking to `()` would require pulling in `ghc-prim` as a dependency. So we just make `()` monospaced instead of linking to it.